dev_delete_all.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# this just deletes everything under /manifests,
+# but tries to space them out a bit to avoid errors.
+# in the end, it creates a custom resource to kick
+# the operator into action
+
+FILE=examples/cr.yaml
+echo "creating ${FILE}"
+oc create -f $FILE
```

Comment:
delete??

Reply:
quick copy paste, haven't tested that one yet.
obviously :)